### ISSUE TITLE
Dragon Accessibility issue with DOB field

### ISSUE THREAD

**Issue by alexnewmannn** [alphagov/govuk-frontend#1101]
We've had a report from someone in the DWP accessibility team about Dragon users being unable to populate any of the 3 date fields, this has been reported in our prototype, another team's service and on the design system site itself.
The user was not able to do any dictation in these fields, but was fine with generic text fields and fields like sort code and account number. The user reported trouble navigating to the fields as they "are not labelled". The end result had to be using a "dictation box" which is designed for when a field will not accept text. It works but is tedious. 

First thought was whether Dragon was seeing something strange about the "pattern" attribute which I'll have an investigate with.

Anyone had any experience with this or got any ideas?

**Comment by joelanman** [comment]
For the benefit of others looking into this, you can see the Date Input component here:

https://design-system.service.gov.uk/components/date-input/

**Comment by selfthinker** [comment]
This is a general issue that Dragon doesn't play well with number fields.
I have reported this to Nuance in September last year. Unfortunately their bug tracking system is not open, but just in case, the reference number is: 170922-000443.

So, this is independent of the Design System as you can see in this [reduced test case](http://output.jsbin.com/munuvub) which uses basic unstyled number fields.

Back then we tested with Dragon 12 and 13, and 13 was worse than 12 in that respect (and some others). I tested Dragon 15 a couple of days ago and found that this has improved a bit: Although it still doesn't work with IE11, it now works with Chrome. (I couldn't test in Firefox because their plugin is not compatible with Quantum.)

**Comment by alexnewmannn** [comment]
Wonder if this would be helpful to add to the documentation for the date input in the design system? So people are aware of this limitation (I know it isn't strictly the component's issue but feels like it should be something we are all aware of)

**Comment by 36degrees** [comment]
We could consider using `input type="text"` instead of `input type="number"` given the other issues we know arise when using `type="number"`.

**Comment by 36degrees** [comment]
Verified that this was fixed by #1700 using Dragon 15 with Internet Explorer 11 on Windows 10.

I checked out and ran 5a3e7719d7b44505c9a8389e20948a33ac32d0ee and tried the following using the [date input example](http://govuk-frontend-review.herokuapp.com/components/date-input/preview):

- Said "Click 'Day'"
- Dragon showed the tooltip 'Please say that again' – the input was not selected.
- Manually focused the day input by clicking into it
- Said "Numeral 5"
- Dragon showed the tooltip 'Please say that again' – nothing was entered in the text box

When testing against the master branch (1d2fcb655361b52efcd851b7b9da3b0e77f05c83) using the same example:

- Said "Click 'Day'"
- Dragon focused the day input
- Said "Numeral 5"
- Dragon entered '5' in the day input

The same steps worked for the month and year.
